### DISCUSSION TITLE
looking to make some bootstraps tweaks

### DISCUSSION THREAD

**Discussion by vespino** [twbs/bootstrap#36206] [Q&A]
[](https://stackoverflow.com/posts/71941181/timeline)[](https://i.stack.imgur.com/dcziH.png)[](https://i.stack.imgur.com/a7Ilw.png)I'm putting together a bootstrap layout and I have some tweaks I would like to do:

- I have a menu consisting of logo, 3 menu items, a button. The logo is positioned left, the button right. I would like to have the menu items centred relatively to the page instead of to logo/button:

![dcziH](https://user-images.githubusercontent.com/6991325/164381233-1b2c95c1-fa7c-4264-ba9b-6699e8afe22f.png)

- The menu turns into an offcanvas menu below lg. I would like to have the dropdown menu open in the menu instead of having to open it manually (maybe remove the toggle?):

![a7Ilw](https://user-images.githubusercontent.com/6991325/164381248-d5e5f7a2-1bfb-496e-8eb2-286baa52e0f1.png)

- At the bottom I have a navbar that is sticky using fixed-bottom. I would like to have it sticky when there is not enough content to fill the page, but only on greater than md. This results in the navbar becoming visible on smaller devices when the user scrolls all the way to the bottom of the page and larger screens always show a fixed navbar.

- I would like the top navbar to overlap the bottom navbar

![Schermafbeelding 2022-04-21 om 07 33 44](https://user-images.githubusercontent.com/6991325/164381453-4b950a93-9f07-4cb9-a17c-1e959f4ff335.png)

Any pointers?

**Comment by gerardnico** [comment]
Check the page header examples on how to center item.

https://getbootstrap.com/docs/5.0/examples/headers/

**Comment by gerardnico** [comment]
Point 3, this is custom JavaScript with a condition. Css is not really good with condition like that.
